<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_main</class>
 <widget class="QWidget" name="login_main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mirror</string>
  </property>
  <widget class="QLabel" name="lbl_bk">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mirror.qrc">:/ui/Resources/ui/l_1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_mirror">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>7</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(179, 179, 179);</string>
   </property>
   <property name="text">
    <string>mirror</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_select">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>454</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/ui/Resources/ui/l_s_1.png);
}
QPushButton:hover{
    border-image: url(:/ui/Resources/ui/l_s_2.png);
}
QPushButton:pressed {
	border-image: url(:/ui/Resources/ui/l_s_3.png);
} 
QPushButton:disabled {
	border-image: url(:/ui/Resources/ui/l_s_4.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>457</y>
     <width>45</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
	border-image: url(:/ui/Resources/ui/l_st_1.png);
}
QPushButton:hover {
	border-image: url(:/ui/Resources/ui/l_st_2.png);
} 
QPushButton:pressed {
	border-image: url(:/ui/Resources/ui/l_st_2.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>486</y>
     <width>116</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
	background-image: url(:/ui/Resources/ui/l_c_1.png);
}
QPushButton:hover {
	background-image: url(:/ui/Resources/ui/l_c_2.png);
} 
QPushButton:pressed {
	background-image: url(:/ui/Resources/ui/l_c_2.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delect">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>506</y>
     <width>116</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
	background-image: url(:/ui/Resources/ui/l_d_1.png);
}
QPushButton:hover {
	background-image: url(:/ui/Resources/ui/l_d_2.png);
} 
QPushButton:pressed {
	background-image: url(:/ui/Resources/ui/l_d_2.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_recover">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>526</y>
     <width>116</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
	background-image: url(:/ui/Resources/ui/l_r_1.png);
}
QPushButton:hover {
	background-image: url(:/ui/Resources/ui/l_r_2.png);
} 
QPushButton:pressed {
	background-image: url(:/ui/Resources/ui/l_r_2.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>547</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
	background-image: url(:/ui/Resources/ui/l_q_1.png);
}
QPushButton:hover {
	background-image: url(:/ui/Resources/ui/l_q_2.png);
} 
QPushButton:pressed {
	background-image: url(:/ui/Resources/ui/l_q_2.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2_select">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>682</x>
     <y>454</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/ui/Resources/ui/l_s_1.png);
}
QPushButton:hover{
    border-image: url(:/ui/Resources/ui/l_s_2.png);
}
QPushButton:pressed {
	border-image: url(:/ui/Resources/ui/l_s_3.png);
} 
QPushButton:disabled {
	border-image: url(:/ui/Resources/ui/l_s_4.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1_level">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1_voc">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>551</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_2_name">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>494</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_2_level">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>523</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_2_voc">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>552</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1_role">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_2_role">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mirror.qrc"/>
 </resources>
 <connections/>
</ui>
